libs/scripts/preview/preview.js

```diff
@@ -0,0 +1,296 @@
+/*
```

Comment:
FYI - This file is a port from Franklin and still has work that needs to be done to make it compatible with Milo.  For now it's ok to ignore this file